fhir/resource_structure_dev/research_subject.json

```diff
@@ -0,0 +1,28 @@
+{
+  "resourceType" : "ResearchSubject",
+  "id" : "example_research_subject",
+  "meta" : {
+    "profile" : [
+      "our_url_here"
+    ]
+  },
+  "status" : "active",
+  "study": {"reference": "ResearchStudy/<id>"},
+  "subject" : {
+    "reference" : "Patient/<id>"
+  },
+  "progress": {
+    "subjectState": {
+      "coding": [
+        {
+          "system": "http://hl7.org/fhir/ValueSet/research-subject-state",
+          "code": "on-study"
+        }
+    ],
+  "text": "On study"
+    }
+  },
+  "period": {
+    "start": "<enrolment date>"
+  }
```

Comment:
we can also add actualComparisonGroup (ARM/ARMCD from SDTM) - mainly used to distinguish patients who only had follow up data collected rather than hospital data from their initial admission.

We can also link to the Consent resource which can be used where studies explicitly indicate consent was taken